src/styles/stars.css

```diff
@@ -0,0 +1,25 @@
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background: url(../images/star.svg) no-repeat center;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
```

Comment:
you should create modifiers for all possible numbers of stars, just like you did in stars task